api-spec.md

```diff
@@ -276,14 +276,13 @@ These parameters and fields are reserved for the Sort extension.
 | --------- | ----------------- | ----------- |
 | sort      | string \| \[Sort] | Placeholder parameter for [API Sort Extension](extensions/sort/README.md). |
 
-### Query Extension
+### Filter Extension
```

Comment:
Might want to keep Query Extension description here as well, but marked as deprecated

Reply:
added a note here.